src/Runner.Worker/ExecutionContext.cs

```diff
@@ -846,6 +848,20 @@ public void QueueAttachFile(string type, string name, string filePath)
             _jobServerQueue.QueueFileUpload(_mainTimelineId, _record.Id, type, name, filePath, deleteSource: false);
         }
 
+        public void QueueResultsSummaryFile(string type, string name, string filePath, string stepId)
```

Comment:
do we need to pass type any more since it's always a summery type?

Reply:
Yeah we should be able to get rid of those types.  The function should only care about summaries.